lib/osut/utils.rb

```diff
@@ -1690,21 +1762,72 @@ def coolingTemperatureSetpoints?(model = nil)
   end
 
   ##
-  # Validates if model has zones with HVAC air loops.
+  # Validates whether space is a vestibule.
   #
-  # @param model [OpenStudio::Model::Model] a model
+  # @param space [OpenStudio::Model::Space] a space
   #
-  # @return [Bool] true if model has one or more HVAC air loops
+  # @return [Bool] returns true if vestibule
   # @return [Bool] false if invalid input (see logs)
-  def airLoopsHVAC?(model = nil)
+  def vestibule?(space = nil)
+    # INFO: OpenStudio-Standards' "thermal_zone_vestibule" criteria:
+    #   - zones less than 200ft2; AND
+    #   - having infiltration using Design Flow Rate
+    #
+    #   github.com/NREL/openstudio-standards/blob/
+    #   86bcd026a20001d903cc613bed6d63e94b14b142/lib/openstudio-standards/
+    #   standards/Standards.ThermalZone.rb#L1264
+    #
+    # This (unused) OpenStudio-Standards method likely needs revision; it would
+    # return "false" if the thermal zone area were less than 200ft2. Not sure
+    # which vintage of 90.1 relies on a 200ft2 threshold (2010?); 90.1 2016
+    # doesn't. Yet even fixed, the method would nonetheless misidentify as
+    # "vestibule" a small space along an exterior wall, such as a semi-heated
+    # storage space.
+    #
+    # The code below is intended as a simple short-term solution, basically
+    # relying on AdditionalProperties, or (if missing) a "vestibule" substring
+    # within a space's spaceType name (or the latter's standardsSpaceType).
+    #
+    # Alternatively, some future method could infer its status as a vestibule
+    # based on a few basic features (common to all vintages):
+    #   - 1x+ outdoor-facing wall(s) holding 1x+ door(s)
+    #   - adjacent to 1x+ 'occupied' conditioned space(s)
+    #   - ideally, 1x+ door(s) between vestibule and 1x+ such adjacent space(s)
+    #
+    # An additional method parameter (i.e. std = :necb) could be added to
+    # ensure supplementary Standard-specific checks, e.g. maximum floor area,
+    # minimum distance between doors.
+    #
+    # Finally, an entirely separate method could be developed to first identify
+    # whether "building entrances" (a defined term in 90.1) actually require
+    # vestibules as per specific code requirements. Food for thought.
```

Comment:
This is tested and works, but additional features could be added (namely to first determine if vestibules are indeed required by code).